starfish/core/_display.py

```diff
@@ -258,13 +265,21 @@ def display(
         # adjust z-size
         sizes[:, 4] *= z_multiplier
 
-        viewer.add_markers(coords=coords, face_color="red", edge_color="red", symbol="ring",
-                           size=sizes * radius_multiplier, n_dimensional=True)
-
-    if new_viewer:
-        window.show()
-
-        if not INTERACTIVE:
-            app.exec()  # create blocking process to persist windows
+        viewer.add_markers(
```

Comment:
now `add_points`

Reply:
also there is no longer a keyword arg `coords`, instead the first arg is the just data array, (can still be a variable named coords)